tests/__init__.py

```diff
@@ -40,7 +38,7 @@
             # Python < 3.8
             _orig_path = os.environ.get('PATH', '')
             os.environ['PATH'] = _orig_path + ';' + _dll_path
-            import openslide
+            import openslide  # noqa: F401  module-imported-but-unused
```

Comment:
Does the other import above also need this?

Reply:
weirdly enough, no